hazelcast/src/test/java/com/hazelcast/mapreduce/MapReduceTest.java

```diff
@@ -44,7 +44,7 @@
 @Category(QuickTest.class)
 @TestProperties(maxThreadsCalculatorClass = MapReduceTestThreads.class)
 @SuppressWarnings("unused")
-public class MapReduceTest
+    public class MapReduceTest
```

Comment:
Hmmm. This  is accidentally appeared space.
Do you think we need to fix it ? ))

FindBugs and checkstyle are ok with it.